### ISSUE TITLE
Invoices - Choose PDF-Templates per record

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#860]
I would like to propose an enhancements for how Invoice module and PDF templates interact:
1. Define which PDF-template to use in each invoice record. A field already exists in the invoice module "Invoice Templates" that would probably be able to be used for this.
2. When choosing action "Print as PDF" or "Email PDF" on a specific record, only show the templates that were defined in the field "Invoice Templates" of respective invoice record. If none was selected, show all available invoice PDF templates. If one was selected don't show the PDF template selector popup -> create PDF with defined PDF template...
3. Allow the bulk printing of PDFs from Invoice List view (as is possible in contact/accounts via "generate letter"). Field "invoice templates" defines for each respective invoice record which pdf template is chosen. If more than one template is chosen for one record, create both pdfs in sequence.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Invoices - Choose PDF-Templates per record](https://trello.com/c/FykjmZCi/124-invoices-choose-pdf-templates-per-record)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
